xml/http_servers.xml

```diff
@@ -1685,11 +1687,12 @@
   <fingerprint pattern="^NS_(\d\.\d)$">
     <description>Citrix NetScaler</description>
     <example>NS_6.1</example>
-    <param pos="0" name="os.vendor" value="Citrix"/>
-    <param pos="0" name="os.family" value="NetScaler"/>
-    <param pos="0" name="os.device" value="Network Management Device"/>
-    <param pos="0" name="os.product" value="NetScaler"/>
-    <param pos="1" name="os.version"/>
+    <param pos="0" name="service.vendor" value="Citrix"/>
+    <param pos="0" name="service.family" value="NetScaler"/>
+    <param pos="0" name="service.device" value="Network Management Device"/>
+    <param pos="0" name="service.product" value="NetScaler"/>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:citrix:netscaler:{service.version}"/>
```

Comment:
FYI @brudis-r7 - This fingerprint has changed quite a bit.

Reply:
OS fields need to be re-added. Add service too, but we can't remove OS without breaking things.